src/game_state/customize_entity_comms_inspection_state.cc

```diff
@@ -12,9 +12,7 @@ extern void set_next_game_state(game_state *s);
 
 extern void warp_mouse_to_center_screen();
 extern bool window_has_focus();
-extern void new_imgui_frame();
-
-extern ImGuiContext *default_context;
+extern ship_space *ship;
```

Comment:
what happened here?

Reply:
bad "git add -p". I'm adding the extern ship, but it wasn't really supposed to be in this hunk. Is benign.